lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset.py

```diff
@@ -582,6 +595,18 @@ def exec_module(self, **kwargs):
                     differences.append('overprovision')
                     changed = True
 
+                nicConfigs = vmss_dict['properties']['virtualMachineProfile']['networkProfile']['networkInterfaceConfigurations']
+                backend_address_pool = nicConfigs[0]['properties']['ipConfigurations'][0]['properties'].get('loadBalancerBackendAddressPools', [])
+                if (len(nicConfigs) != 1 or len(backend_address_pool) != 1):
+                    support_lb_change = False  # Currently not support for the vmss contains more than one loadbalancer
+                    self.module.warn('Not support to update loadbalancer for the vmss contains more than one loadbalancers.')
```

Comment:
I think this warning text should be reworded as it will be displayed to customers:
Maybe: "Updating more than one load balancer on VMSS is currently not supported"
Just this one and we can merge